cypress/integration/pages/frontPage/tests.js

```diff
@@ -168,42 +185,19 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
-        it('should contain Useful Links if usefulLinks block data exists', () => {
+
+        it('should contain Useful Links if valid usefulLinks block data exists', () => {
           cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
             ({ body }) => {
               const pageData = body.content.groups;
-              const usefulLinks = pageData.find(data => {
-                return data.type === 'useful-links';
-              });
-
-              const contentTypes = [
-                'Text',
-                'Feature',
-                'Audio',
-                'Video',
-                'Gallery',
-                'Guide',
-              ];
-
-              let usefulLinksItems = [];
-              if (usefulLinks) {
-                usefulLinksItems = usefulLinks.items.filter(item => {
-                  return (
-                    item.assetTypeCode === 'PRO' &&
-                    contentTypes.includes(item.contentType)
-                  );
-                });
-              }
-
-              const isValidUsefulLinks =
-                usefulLinks &&
-                'strapline' in usefulLinks &&
-                usefulLinksItems.length;
-
-              if (isValidUsefulLinks) {
-                //  We include the hasStrapline as we don't render Useful Links
-                //  if we don't receive a strapline in the data
-                cy.get('[aria-labelledby="Useful-links"]').should('be.visible');
+              if (isValidUsefulLinks(pageData)) {
+                cy.get('[data-e2e="useful-links"]')
+                  .eq(0)
+                  .within(() => {
```

Comment:
LGTM, but this could be improved checking if they link is within a `ul li` when there is more than one item, and within a `div` when there is just one.

Reply:
I thought of that, but was of the opinion that this is checked in the unit tests (https://github.com/bbc/simorgh/blob/c35b8d74327bc7cca723f58b33080e66ec6a2d7b/src/app/containers/FrontPageSection/UsefulLinks/index.test.jsx#L7); so the integration test should check if the Useful links appear when they are supposed to, and do not appear when they shouldn't